homework1/src/RemoveDuplicates.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+
+public class RemoveDuplicates {
+    public static void main(String[] args) {
+        Integer[] numbers = {7, 10, 11, 15, 2, 3, 3, 10, 11, 20};
+        Arrays.sort(numbers, Collections.reverseOrder());
+
+        System.out.print(Arrays.toString(numbers));
```

Comment:
it will be more readable if there will be a new line between outputs
in this case you can use `System.out.println()`
ln in the end means new line

OR else you can use `System.out.print(something + "\n");`

Reply:
Fixed in commit: f1536a4